azure-sdk-testutils/devtools_testutils/mgmt_settings_fake.py

```diff
@@ -22,7 +22,7 @@
 # Read for details of this file:
 # https://github.com/Azure/azure-sdk-for-python/wiki/Contributing-to-the-tests
 
-def get_credentials():
+def get_credentials(**kwargs):
```

Comment:
I don't think that's necessary? Since I will not inject data in the credentials anyway?

Reply:
I did it because we need to inject a custom resource into a ServicePrincipalCredentials in the real_creds, which then errors in playback....